pkgs/top-level/all-packages.nix

```diff
@@ -23912,15 +23912,16 @@ in
 
   vimpc = callPackage ../applications/audio/vimpc { };
 
-  wrapNeovim = callPackage ../applications/editors/neovim/wrapper.nix { };
-
+  wrapNeovimUnstable = callPackage ../applications/editors/neovim/wrapper.nix { };
```

Comment:
It would be nice to have a small comment on what "Unstable" exactly means ("If you use this API in your config, it might break").

Reply:
I've added the following comment:
```
 # this is a lower-level alternative to wrapNeovim conceived to handle
+  # more usecases when wrapping neovim. The interface is being actively worked on
+  # so expect breakage. use wrapNeovim instead if you want a stable alternative
```